File: CPP/1012.cpp
#include <iostream>
#include <iomanip>
//Area
using namespace std;

int main(){
	double a, b, c;
	cin >> a >> b >> c;
	std::cout << std::fixed << std::setprecision(3) << "TRIANGULO: " << (a*c)/2 << "\nCIRCULO: " << 3.14159*(c*c) << "\nTRAPEZIO: " << ((a+b)*c)/2 << "\nQUADRADO: " << b*b << "\nRETANGULO: " << a*b << endl;

	return 0;
}

File: uriOnlineJudge/python/1012.py
a, b, c = input().split()
a, b, c = [float(a), float(b), float(c)]
print("TRIANGULO: %.3f\nCIRCULO: %.3f\nTRAPEZIO: %.3f\nQUADRADO: %.3f\nRETANGULO: %.3f"%((a*c)/2,3.14159*(c*c),((a+b)*c)/2,b*b,a*b))